resources/modes/print/print.html

```diff
@@ -65,6 +65,14 @@
   <div id="shadow"></div>
 </div>
 
+  <div class="modal-dialog hidden" id="processing-dialog">
+    <div>
+    <p>
+      Processing SVG.
```

Comment:
Don't forget: All text should live in translation strings. Don't worry about the other languages, I'll run the language parser right before release. Some ideas for text here:
```
[interesting animated gif here]
Processing seems to be taking longer than expected, please wait.
```
@oskay may want to chime in on this one.